py/src/braintrust/logger.py

```diff
@@ -1942,6 +1942,7 @@ def summarize(self, summarize_scores=True, comparison_experiment_id=None):
 
         return ExperimentSummary(
             project_name=self.project.name,
+            experiment_id=self.id,
```

Comment:
fyi @ankrgyl: note an interesting consequence of this is that experiment summary with `summarize_scores=False` will now require a network roundtrip in order to resolve the id.

Reply:
Oh interesting. Didn't it require one before to get the experiment's name?

Reply:
Ah yes good point. Never mind.